internal/packages/packages.go

```diff
@@ -163,6 +163,11 @@ func FindDataStreamRootForPath(workDir string) (string, bool, error) {
 	return "", false, nil
 }
 
+// ReadPackageManifestForPackage reads and parses the package manifest file for the given package.
+func ReadPackageManifestForPackage(packageRoot string) (*PackageManifest, error) {
```

Comment:
Nit: this function name reads a bit awkwardly. How about `ReadPackageManifestFromPackageRoot`?

Reply:
Renamed.